samples/sensor/max30208/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  description: TBD
+  name: TBD
+tests:
+  sample.sensor.max30208:
+    harness: sensor
+    tags: sensors
+    platform_allow: nrf52dk_nrf52832
```

Comment:
This sample seems to be generic enough, so it can run on more boards. Better use `filter: dt_compat_enabled("maxim,max30208")`. See for example `samples/sensor/lps22hh/sample.yaml`.